Tavisca.Bootcamp.LanguageBasics.Exercise1/FixMultiplication.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
+{
+    public class FixMultiplication
+    {
+        public static int FindDigit(string equation)
+        {
+            // a * b = c
+            //Extrct 3 numbers
+
+            var indexOfMultiply = equation.IndexOf('*');
+            var indexOfEqual = equation.IndexOf('=');
+            var multiplier1 = equation.Substring(0, indexOfMultiply);
+            var multiplier2 = equation.Substring(indexOfMultiply + 1, indexOfEqual - indexOfMultiply - 1);
+            var result = equation.Substring(indexOfEqual + 1, equation.Length - indexOfEqual - 1);
+
+            var missingDigitPlaceHolder = multiplier1.Contains('?') ? 1 : multiplier2.Contains('?') ? 2 : 3;    //finding position of '?' in the string
+
+            //Console.WriteLine($"{multiplier1} {multiplier2} {result}")
+
+            // converting string to int
+            int first = 0;
+            int second = 0;
+            int third = 0;
+            if (missingDigitPlaceHolder != 1)
+                int.TryParse(multiplier1, out first);
+
+            if (missingDigitPlaceHolder != 2)
+                int.TryParse(multiplier2, out second);
+
+            if (missingDigitPlaceHolder != 3)
+                int.TryParse(result, out third);
+
+            //finding third value
+            int temp = 0;
+            int Ceil_value = 0;
+
+            if (missingDigitPlaceHolder == 1 && second != 0)
+            {
```

Comment:
This if condition has the same logic as in the next if block.. you can reuse the code by writing the code in method